pkg/transfer/image/imagestore.go

```diff
@@ -22,13 +22,16 @@ import (
 
 	"github.com/containerd/typeurl/v2"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/pkg/errors"
```

Comment:
one more... let's not add a new direct import of `pkg/errors` as we don't use it elsewhere in the code base. 

Look like `fmt.Errorf` with a `%w` is favored - https://github.com/search?q=repo%3Acontainerd%2Fcontainerd+%2Freturn+fmt.Errorf.*err%2F&type=code